content/20.pharmaceuticals.md

```diff
@@ -435,100 +435,136 @@ Thus, advances in biologics for COVID-19 illustrate that an understanding of how
+However, it is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
+Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effects in treating COVID-19, but are not the only mechanism by which this drug can interfere with disease progression.
+
+#### Potential and Limitations of High-Throughput Analyses
+
+These examples illustrate that computational research in drug repurposing can provide some valuable insights, but there are significant limitations to these methods.
+The urgency of the COVID-19 pandemic led many research groups to pivot towards this area of research without familiarity with best practices [@doi:10.1039/D0CS01065K]; as a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.
+Furthermore, it's not clear to what extent large-scale computational screening studies, especially those that are hypothesis free, produce insights that wouldn't be gained from smaller experimental screening efforts [@doi:10.1039/D0CS01065K; @doi:10.1126/science.abj9488].
+Fluvoxamine was not one of the Sigma-1 receptor modifying drugs screened for _in vitro_ antiviral activity against SARS-CoV-2 by the original screening study [@doi:10.1038/s41586-020-2286-9], but it was under investigation based on its immunomodulatory potential.
+The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free screening: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].
+Computational analysis correctly identified the Sigma-1 receptor as a protein of interest, but the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported. 
+In comparison, computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
+Despite the concerns it raises about high-throughput screening, it was a data-driven approach that allowed researchers to notice a shared property among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
+In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms are not likely to produce valuable candidates.
```

Comment:
This is pretty strong ("are not likely"). The evidence looks that way... but perhaps different screens in the future will be more valuable? I wonder if there's a way to lean on the sentence that follows. Like, what if it ended with "may not produce valuable candidates."

Reply:
```suggestion
In combination, these findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis or without an emphasis on biological mechanisms may not produce valuable candidates.
```
Thanks @cgreene!